sycl/test-e2e/Matrix/joint_matrix_bf16_fill_k_cache_OOB.cpp

```diff
@@ -0,0 +1,23 @@
+//==--- joint_matrix_bf16_fill_k_cache_OOB.cpp  - DPC++ joint_matrix--------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: matrix
+
+// RUN: %{build} -o %t_gpu.out -ffp-model=precise -DOOB
+// RUN: %if gpu %{ %{run} %t_gpu.out %}
```

Comment:
If test is not intended to run on CPU, then it is better to add "REQUIRES: gpu", to avoid building the test on CPU